helm-chart/renku/requirements.yaml

```diff
@@ -2,27 +2,27 @@ dependencies:
 - name: renku-core
   alias: core
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: 0.12.2
+  version: 0.12.3-93348f9
```

Comment:
these should be latest released versions, no?

Reply:
My thinking was to just take over whatever is on development now, remove development so that people stop updating dependencies there and then figure out the right tagged version for each component in the release-prep PR that should follow shortly after.

Reply:
ok sounds good!